rep-2005.rst

```diff
@@ -0,0 +1,568 @@
+
+
+Change process
+==============
+
+This content is maintained at GitHub in the `REP repo <https://github.com/ros-infrastructure/rep>`_.
+To propose changes, such as adding or removing items, make a pull request (PR) to that repo.
+The ensuing public review and discussion will evaluate the change in light of the purpose and scope explained above.
+The final merge decision on a given PR will be made by the ROS 2 Technical Steering Committee (TSC) by a simple voting process:
+
+* Each TSC member organization may vote on a PR by commenting with a score according to the `REP voting guidelines <https://www.ros.org/reps/rep-0010.html#voting-scores>`_.
+* A TSC member may update their vote (e.g., following changes or discussion) with a new comment while the PR remains under review (i.e., before it has been merged); the new vote takes the place of the old one.
+* The PR will be merged if and when the sum of TSC member vote values is **>= 3** (votes of `+0` and `-0` are treated equally as 0 when computing the sum).
+* The PR will be declined if it becomes numerically impossible based on existing votes to meet the minimum vote threshold.
+* The proposer will be encouraged to withdraw a PR if after a significant amount the time the minimum vote threshold is still not met.
+
+For example, if a PR garners the following 6 votes from TSC members: `+1`, `+1`, `-1`, `+0`, `+1`, `+1`, then the sum would be 3 and the PR would be merged.
+
+The minimum vote threshold necessary to merge a PR may be changed by a majority vote of the TSC.
+
+
+Guidance on changes
+===================
+
+We wish for this list to grow steadily over time, but not to become an indiscriminate collection of all ROS 2 software.
+Our goal with this list is **curation**: we want to ensure that the contents are meaningful to the developer and user community.
+All else being equal, we prefer to include packages that:
+
+* have clear evidence of reuse (e.g., declared dependencies elsewhere in ROS 2, stars or forks at GitHub, public user testimonials); and/or
+* represent substantially new and important features that are reasonably expected to come into wide use but are not yet ready; and/or
+* are demonstrably of high quality (e.g., via conformance with REP 2004).
+
+We prefer not to include packages that meet none of these criteria.
+
+
+Content
+=======
+
+The common packages may vary among distros; unless otherwise annotated below, each package in the list is **available since Foxy Fitzroy**.
+When planning starts for a new distro, the package list from the previous distro will be copied in as a starting point.
+All items on the list for the next distro should already be released into the rolling distro; as a bootstrapping exception, items on the list for Foxy may take some time to be released.
+
+* Buildsystem, package index and tooling around
```

Comment:
Has it been considered adding colcon packages in this list ?
I understand the scope of the colcon project is larger than ROS 2 but parts of it are tailored to ROS and most (all?) ROS 2 users rely on it for their development

Reply:
I defer to @dirk-thomas on that question.